### DISCUSSION TITLE
Package babel Error: You haven't specified a language option.

### DISCUSSION THREAD

**Discussion by robline** [jgm/pandoc#9880] [Q&A]
I am trying to generate a pdf from a webpage. I'm on Ubuntu 18.04 using pandoc 3.2-1. I've installed TeXLive and have all the languages installed for babel. I did a fresh install of openbabel. When I run any of the commands below, I get the same error. How can I generate a pdf without getting this error?  I have also tried en-GB and en-US as lang options.

```
Error producing PDF.
! Package babel Error: You haven't specified a language option.

See the babel package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.478 ...ry to proceed from here, type x to quit.}

```

`pandoc -f html -t pdf http://www.columbia.edu/~fdc/sample.html -o test.pdf `

`pandoc -V lang=en -f html -t pdf http://www.columbia.edu/~fdc/sample.html -M lang:en -o test.pdf `

`pandoc -V lang=english -f html -t pdf http://www.columbia.edu/~fdc/sample.html -M lang:english -o test.pdf`

`pandoc -V babel-lang=en -f html -t pdf http://www.columbia.edu/~fdc/sample.html -M babel-lang:en -o test.pdf `

`pandoc -V babel-lang=english -f html -t pdf http://www.columbia.edu/~fdc/sample.html -M babel-lang:english -o test.pdf`

`pandoc -V babel-lang=en -f html -t pdf http://www.columbia.edu/~fdc/sample.html -M babel-lang:en -o test.pdf --pdf-engine=xelatex`

`pandoc -V babel-lang=english -f html -t pdf http://www.columbia.edu/~fdc/sample.html -M babel-lang:english -o test.pdf --pdf-engine=xelatex`

**Comment by jgm** [comment]
Tried the first three, and all of them worked fine on my system. (Didn't try others.)
To diagnose this, use a minimal input file (as small as you can make it and still get the error).  Then add `--verbose` and you'll see the intermediate LaTeX. Look for lines with babel.
